### ISSUE TITLE
contributors widget should have parameter for "dofollow" links

### ISSUE THREAD

**Issue by DreadKnight** [opencollective/opencollective#5968]
## User story 
_As <persona>, I can <what?> so that <why?>_
Contributors want a clickable banner on our website. Problem is that most want the link to be "dofollow", while the "Financial contributors widget" only provides "nofollow" links. Placing banners manually on the project webpages is a total no in most cases, as it takes work and too hard to maintain who comes and leaves.

## Best solution for this problem
Would be best if the "Financial contributors widget" would have a new parameters that will allow to overwrite the default "nofollow" types a link into false, making them "dofollow" by actually removing the rel attribute from the output code.

## MVP
Customizable widget with new parameter that will allow "dofollow" links.

## Metrics 
This should allow for increased number of sponsors, easier management and for more options regarding tier levels.

## Documentation 
New parameter "follow" that defaults to "no-follow", setting it to "false" will turn links clickable sponsor links to become "dofollow".

**Comment by Betree** [comment]
Some documentation about the origin of this change can be found in https://github.com/opencollective/opencollective/issues/3461 / https://github.com/opencollective/opencollective/issues/3462. Many collectives don't want that to be the default as it attracts dubious sponsors (e.g., casinos), but I'm not against having that as an opt-in option.

**Comment by DreadKnight** [comment]
@Betree Yeah, I totally get it, though some more options and customization are always nice, especially since the widget already accepts some parameters. I'm glad that you're for this as well, it's a simple little feature that can have a big positive impact overall.

**Comment by DreadKnight** [comment]
> Some documentation about the origin of this change can be found in #3461 / #3462. Many collectives don't want that to be the default as it attracts dubious sponsors (e.g., casinos), but I'm not against having that as an opt-in option.

Any ETA on this? I've lost like 10 sponsors so far because of it, as most of them seek follow up links and banners for SEO purposes but they're shy to talk about it, only a few messaged me to ask about it. I need to add banners manually as well it seems, which is a total nightmare to manage, especially if things will eventually scale (hopefully). This stuff is super important to sponsors (follow up links) and without money, most collectives can't do too much.

**Comment by Betree** [comment]
The parameter needs to be added in https://github.com/opencollective/opencollective-images/blob/8ebc2d9d2b6fbdbf146ac92ccf01cf2c46c13eae/src/server/lib/svg-banner.js. 

This issue hasn't been prioritized by the core team yet, but contributions are welcome. 

Alternatively, you can use our public GraphQL API to fetch the necessary data and build your own widget.

**Comment by DreadKnight** [comment]
> The parameter needs to be added in https://github.com/opencollective/opencollective-images/blob/8ebc2d9d2b6fbdbf146ac92ccf01cf2c46c13eae/src/server/lib/svg-banner.js.
> 
> This issue hasn't been prioritized by the core team yet, but contributions are welcome.
> 
> Alternatively, you can use our public GraphQL API to fetch the necessary data and build your own widget.

@Betree  Understood. This should be prioritized, as most backers/sponsors care about having their clickable avatars showcased and even care about SEO stuff (even though I think it's rather silly). I get messaged when avatars aren't showing up and from time to time regarding why are the links "no-follow", threatening to stop contributing. Sometimes I have to manually add duplicate of their clickable avatar just to please them. The widget is what sets OC apart from any competition and backers/sponsors tend to care more about SEO rather then transparency, sadly...

**Comment by DreadKnight** [comment]
Alrighty, I've created an issue in my repo to increase awareness, providing more details and also offering a bounty 🪙 in our token.
You can check it out over here -> https://github.com/FreezingMoon/AncientBeast/issues/2640

**Comment by anshgoyalevil** [comment]
Submitted a PR: https://github.com/opencollective/opencollective-images/pull/616
